insights/parsers/cron_daily_rhsmd.py

```diff
@@ -0,0 +1,34 @@
+"""
+CronDailyRhsmd - file ``/etc/cron.daily/rhsmd``
+===============================================
+"""
+
+from insights.core import Scannable
+from insights.core.plugins import parser
+from insights.specs import Specs
+
+
+@parser(Specs.cron_daily_rhsmd)
+class CronDailyRhsmd(Scannable):
+    """
+    Parse the ``/etc/cron.daily/rhsmd`` file.
+
+    Sample input::
+
+        config=$(grep -E "^processTimeout" /etc/rhsm/rhsm.conf | grep -Po "[0-9]+")
+        rhsmd_timeout=$config
+        abc=$config
+
+    Examples:
+
+        >>> from insights.tests import context_wrap
+        >>> from insights.parsers.cron_daily_rhsmd import CronDailyRhsmd
+        >>> CronDailyRhsmd.collect('config_lines', lambda n: n if "$config" in n else "")
+        >>> CronDailyRhsmd.any('one_config_line', lambda n: n if "$config" in n else "")
+        >>> rhsmd=CronDailyRhsmd(context_wrap(RHSMD_1))
```

Comment:
@huali027 thanks for making the changes in your parser, use of `Scannable` was a good call.  This example is a little misleading in that someone using this parser would not use it in this way with `context_wrap` since that function is only for testing.  I'd recommend you add lines 24-27 to your doc test to create the `rhsmd` object and then leave lines 26-27 here, but comment them out and add a reference to the `Scannable` class which provides more extensive documentation and examples.

Reply:
@bfahr, sure, I'll update the PR, thanks.